src/main/java/com/ua/accommodation/service/impl/UserServiceImpl.java

```diff
@@ -36,7 +36,6 @@ public UserResponseDto updateRoles(Long userId, UserUpdateRoleDto updateRoleDto)
     @Override
     public UserResponseDto updateProfile(Long userId, UserUpdateProfileDto updateProfileDto) {
         User user = getUserById(userId);
-        user.setEmail(updateProfileDto.getEmail());
```

Comment:
why do we need this change?

Reply:
because email change need log out. It look like operation for another controller